loadtest/bulkload.go

```diff
@@ -412,15 +413,27 @@ func LoadPosts(cfg *LoadTestConfig, dbendpoint string) {
 	numPostsPerChannel := int(math.Floor(float64(cfg.LoadtestEnviromentConfig.NumPosts) / float64(cfg.LoadtestEnviromentConfig.NumTeams*cfg.LoadtestEnviromentConfig.NumChannelsPerTeam)))
 
 	statementStr := "INSERT INTO Posts (Id, CreateAt, UpdateAt, EditAt, DeleteAt, IsPinned, UserId, ChannelId, RootId, ParentId, OriginalId, Message, Type, Props, Hashtags, Filenames, FileIds, HasReactions) VALUES "
-	for i := 0; i < 100; i++ {
-		statementStr += "(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),"
+	if driverName == "mysql" {
+		for i := 0; i < 100; i++ {
+			statementStr += "(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?),"
+		}
+	} else if driverName == "postgres" {
+		for i := 0; i < 100; i++ {
+			statementStr += fmt.Sprintf(
+				"($%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d, $%d),",
```

Comment:
`github.com/lib/pq` doesn't support `?`. I didn't look much harder for other Postgres drivers, but opinions welcome :)